examples/basic_example/basic_example.py

```diff
@@ -71,28 +71,49 @@
 from oemof.solph import EnergySystem
 from oemof.solph import Model
 from oemof.solph import buses
-from oemof.solph import components as cmp
+from oemof.solph import components as solver_components
 from oemof.solph import create_time_index
 from oemof.solph import flows
 from oemof.solph import helpers
 from oemof.solph import processing
 from oemof.solph import views
 
+BATTERY_STORAGE = "battery_storage"
+
+
+def get_data_from_file_path(file_path: str) -> pd.DataFrame:
```

Comment:
We do not use type hints in other locations, but I like them and we have to start somewhere.